dags/operators/email_schema_emr_spark_operator.py

```diff
@@ -0,0 +1,55 @@
+import boto3
+import re
+from airflow.exceptions import AirflowException
+from airflow.utils.decorators import apply_defaults
+from airflow.utils.email import send_email
+from difflib import unified_diff
+from .emr_spark_operator import EMRSparkOperator
+
+class EmailSchemaEMRSparkOperator(EMRSparkOperator):
```

Comment:
I don't think we should conflate responsibilities for operators like this -- seems like it'd make more sense to make this a separate task that runs after main_summary. Consider the case where we fail to send an email because the SMTP service is down, or somesuch -- we'd re-run the entirety of main_summary for that. Also, if we make the email task separate we can inherit from the built-in EmailOperator which seems like a more obvious lineage.

Reply:
fixed